cylc/flow/dbstatecheck.py

```diff
@@ -109,39 +128,72 @@ def get_remote_point_format_compat(self):
             return row[0]
 
     def state_lookup(self, state):
-        """allows for multiple states to be searched via a status alias"""
+        """Allows for multiple states to be searched via a status alias."""
         if state in self.STATE_ALIASES:
             return self.STATE_ALIASES[state]
         else:
             return [state]
 
     def workflow_state_query(
-            self, task, cycle, status=None, message=None, mask=None):
-        """run a query on the workflow database"""
+        self,
```

Comment:
I'm slightly questioning whether this is attached to the correct object: Should this perhaps be attached to the workflow database or something?

Reply:
Not sure I understand the question @wxtim ??  `self` is an object that knows which DB to connect to once the query has been formulated based on the command options.

Reply:
Yes. That's correct. I missed that this is semi-standalone.